Sources/SwiftHtml/Attributes/Global.swift

```diff
@@ -106,6 +119,7 @@ public extension Tag {
     
     /// Removes an array of class values if the condition is true
     func `class`(remove values: [String], _ condition: Bool = true) -> Self {
+        guard condition else { return self }
```

Comment:
I could be wrong... but it seems like ```class``` can get deleted even if the condition is false